README.md

```diff
@@ -1 +1,44 @@
-# interview-eng-i
\ No newline at end of file
+# interview-eng-i
+
```

Comment:
It would be useful to provide context of the test and a list of technologies used ahead of the getting started section.